packages/react-lowcode/src/codegen/generation/generators/list/mui-datatable-generator.ts

```diff
@@ -262,33 +261,56 @@ export default class MuiDataTableGenerator implements TableGenerator
         let ast = createAst(template);
 
         if (ast) {
-          const templateParenthesizedExpression = find<ParenthesizedExpression>(ast, (node: Node) => {
-            return ts.isParenthesizedExpression(node);
-          });
+          const tableComponentName = this._helper.getComponentName(this._entity);
+          const inputParameterIdentifier = this._helper.getInputParameterIdentifier(this._entity);
 
-          if (templateParenthesizedExpression) {
-            const tableComponentName = this._helper.getComponentName(this._entity);
-            const inputParameterIdentifier = this._helper.getInputParameterIdentifier(this._entity);
-
-            const parenthesizedExpression = factory.createParenthesizedExpression(factory.createJsxSelfClosingElement(
-              factory.createIdentifier(tableComponentName),
-              undefined,
-              factory.createJsxAttributes([factory.createJsxAttribute(
-                inputParameterIdentifier,
-                factory.createJsxExpression(
-                  undefined,
-                  factory.createPropertyAccessChain(
-                    factory.createIdentifier("data"),
-                    factory.createToken(ts.SyntaxKind.QuestionDotToken),
-                    inputParameterIdentifier
-                  )
+          const tableComponentElement = factory.createJsxSelfClosingElement(
```

Comment:
please wrap this code to a helper method/function

Reply:
Was created function: createTableComponentElement (src\codegen\generation\generators\template\template-resolver.ts)